dynamic-layers/meta-oe/modemmanager/files/modemmanager.initd

```diff
@@ -0,0 +1,11 @@
+#!/sbin/openrc-run
+
+supervisor=supervise-daemon
+command=/usr/sbin/ModemManager
+command_args="${modemmanager_opts}"
```

Comment:
`modemmanager_opts` isn't defined anywhere.  To follow what a lot of the other init scripts are doing in this layer, simply add a `modemmanager.confd` that has a default `COMMAND_ARGS` (https://github.com/jsbronder/meta-openrc/blob/master/recipes-appends/oe-core/acpid/acpid.confd#L2) which `command_args` is set to here along with any mandantory arguments (https://github.com/jsbronder/meta-openrc/blob/master/recipes-appends/oe-core/acpid/acpid.initd#L6).